numpy/distutils/ccompiler_opt.py

```diff
@@ -234,7 +234,8 @@ class _Config:
         ppc64le = "VSX VSX2",
         s390x = '',
         armhf = '', # play it safe
-        aarch64 = "NEON NEON_FP16 NEON_VFPV4 ASIMD"
+        aarch64 = "NEON NEON_FP16 NEON_VFPV4 ASIMD",
```

Comment:
Perhaps someone else should confirm this, but AFAIK distutils is deprecated.

Reply:
I have deleted it. And do you mean just at line 237, or should all changes in the this file be abandoned.

Reply:
I have deleted all.